src/main/resources/citizen/templates/document/countyCourtJudgmentByAdmission.html

```diff
@@ -0,0 +1,242 @@
+<html>
```

Comment:
could you rename the file to be `countyCourtJudgmentIssued.html` or something similar please